.gitlab/new-e2e_testing/suse.yml

```diff
@@ -9,6 +9,32 @@
     TEAM: agent-platform
     EXTRA_PARAMS: --osversion $E2E_OSVERS --platform $E2E_PLATFORM --cws-supported-osversion $E2E_CWS_SUPPORTED_OSVERS --major-version $AGENT_MAJOR_VERSION --arch $E2E_ARCH --flavor $FLAVOR
 
+.new-suse-e2e_script_upgrade7:
```

Comment:
Why does suse need a different `upgrade` base job?

Reply:
Was because we don't want to use agent 5 on suse but forgot we can just override `parallel matrix` so not needed fixed 👍